layouts/partials/cookie-consent-modal.html

```diff
@@ -2,17 +2,15 @@
 <div id="cookieModal" class="modal custom fade" role="dialog" data-backdrop="false">
   <div class="modal-dialog modal-xl">
     <div class="modal-content">
-      <!-- <div class="modal-header">
-       <p class="modal-title">Website analytics cookies</p>
-       </div> -->
       <div class="modal-body">
-        <p>We use cookies. <a href="https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage" target="_blank">Google Analytics</a> is used to improve your experience and help us understand site traffic and page usage. <a href="https://www.cookiechoices.org/" target="_blank">Learn more.</a></p>
+        <p>We use cookies. <a href="https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage" target="_blank">Google Analytics</a> is used to improve your experience and help us understand site traffic and page usage.</p>
         <div class="consent-buttons">
-          <button type="button" class="btn btn-sm mr-3 mb-4" onclick="optout()">Opt Out</button>
+          <button type="button" class="btn btn-sm mr-3 mb-4" onclick="learnaboutcookies()">Learn more</button>
           <button type="button" class="btn btn-sm btn-secondary mr-3 mb-4" onclick="acceptcookie()">Accept</button>
           <div class="opt-out">
-            <!--<p>You have the option to refuse the use of cookies by selecting the appropriate settings on your browser, however please note that if you do this, you may not be able to use the full functionality of this website.</p> -->
-            <p>Opt out of Google Analytics by disabling cookies or JavaScript in your browser, or by using the <b><a href="https://tools.google.com/dlpage/gaoptout" target="_blank">opt-out service provided by Google</a></b>.</p>
+            <hr>
+            <p><a href="/about-analytics-cookies/">Learn about analytics cookies and how you can take steps to opt-out from sharing your usage data.</a></p>
+            <button type="button" class="btn btn-sm btn-primary mr-3 mb-4" onclick="optout()">I understand how to opt-out, disable this notice.</button>
```

Comment:
Can we change this to "hide this notice"?

Reply:
done